recipes/blender-mathutils/recipe.yaml

```diff
@@ -0,0 +1,61 @@
+context:
+  name: blender-mathutils
+  version: "3.3.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/m/mathutils/mathutils-${{ version }}.tar.gz
+  sha256: b9371af8149f3f3f05ee165cb583b2df7b3fbc9935406d1c505a5276b8e563af
+  patches:
+    - if: match(python, ">=3.13")
```

Comment:
Why not just apply patches unconditionally? If you write the patch correctly you should be able to do that.